Example/FailedRequestViewController.swift

```diff
@@ -25,10 +25,11 @@ class FailedRequestViewController: UIViewController {
             .didFailProvisionalNavigation
             .observeOn(MainScheduler.instance)
             .debug("didFailProvisionalNavigation")
-            .subscribe(onNext: {(webView, navigation, error) in
+            .subscribe(onNext: { [weak self] webView, navigation, error in
+                guard let strongSelf = self else { return }
```

Comment:
can you do 
```suggestion
                guard let self = self else { return }
```

As available since Swift 4.2 ?

Reply:
Done